c2corg_ui/static/partials/map/map.html

```diff
@@ -13,7 +13,12 @@
     </div>
   </div>
 
-  <button type="button" ng-show="mapCtrl.edit" ng-click="mapCtrl.resetFeature()"
-          class="btn btn-primary btn-xs map-reset-feature" translate>Reset feature</button>
+  <button type="button" ng-show="mapCtrl.canReset()" ng-click="mapCtrl.resetFeature()"
+          class="btn btn-primary btn-xs map-reset-feature"
+          title="{{ 'Reset geometry' | translate }}" translate>Reset geometry</button>
```

Comment:
> I don't think normal users know more about "geometries" than about "features".

I think everybody knows at least the word *geometry*.

> I have not tested with point documents. Does your tool apply to such features?

Yes, you can also delete point geometries.